apps/www/content/docs/installation.mdx

```diff
@@ -36,6 +36,7 @@ Where is your tailwind.config.js located? › tailwind.config.js
 Configure the import alias for components: › @/components
 Configure the import alias for utils: › @/lib/utils
 Are you using React Server Components? › no / yes
+Write configuration to components.json. Proceed? › (Y/n)
```

Comment:
Let's not put this one here. We only want the prompts and not the confirmation.

Reply:
Removed it.